SUBSYSTEMS.md

```diff
@@ -0,0 +1,196 @@
+# RIOT Subsystem Maintainers
+
+This file gives an overview about the maintainers of RIOT subsystems. Whenever
+you submit an issue or a Pull Request, please assign one of the maintainers of
+the affected modules as an assignee in the issue tracker. The maintainer takes
+care of either fixing/merging the issue/PR herself or re-assign the issue to a
+qualified developer.
+
+**Note**: A developer might maintain more than one subsystem and an issue might
+affect more than one module.
+
+# CPU
+## ESP32, ESP8266
+- Gunar Schorcht [gschorcht](https://github.com/gschorcht)
+
+## MSP430
+- Oleg Hahm [@OlegHahm](https://github.com/OlegHahm)
+
+## ARM7
+- Kaspar Schleiser [@kaspar030](https://github.com/kaspar030)
+
+## ARM Cortex-M
+- Hauke Petersen [@haukepetersen](https://github.com/haukepetersen)
+- Thomas Eichinger [@thomaseichinger](https://github.com/thomaseichinger)
+- Nick van IJzenoorn [@DipSwitch](https://github.com/DipSwitch)
```

Comment:
Please don't just 'migrate' over inactive maintainers, start from scratch and let active maintainers fill in what they feel responsible for.

If we 'assign' people this will have to value, they have to actively feel responsible for their subsystem.

Reply:
I kind of created the subsystems from scratch but migrated some of the maintainers. Should I just drop them all and let them add themselves?

Reply:
That sounds like a good idea. Let's also merge this without maintainers in it. We can distribute and parallize the work the better once the file is merged. (I fear otherwise there will be an endless discussion on nitpicking who to add and who not.)

Reply:
Yes, we don't want the situation where people are like "Oh, I don't know why I'm on that list, I don't feel responsible for that subsystem".

There should be *some* commitment that can be expressed by actively editing the list.

I think we can also push to your branch directly.

Reply:
done! I just removed all maintainers from the list. We can merge this one and then ask maintainers to add themselves to the list, as proposed above.